docs/intro.md

```diff
@@ -24,7 +24,7 @@ The Subspace Network is an ambitious layer zero protocol which is the first scal
 Our goal is to bring an extremely low barrier to entry for participating on consensus, As long as you meet the simple requirements below you can get started below.
 
 
-### - [Start Farming with Subspace CLI](/docs/protocol/cli)
+### - [Start Farming with Subspace CLI](../protocol/cli)
```

Comment:
This link switches the `pre-release` version to `latest` for me locally.
If I change it to `(./protocol/cli)` the versioning works as expected. Can you test it too, please?

Reply:
Resolved in commit 97dd5cf & 9a76b88